plugins/dev-create/templates/block/template/test/block_test.mocha.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {assert} from 'chai';
+import {sinon} from 'sinon';
```

Comment:
Does this mean sinon and chai need to be added to the block template's package.json?